src/style.css

```diff
@@ -1,3 +1,92 @@
+@import url('https://fonts.googleapis.com/css2?family=Rubik:wght@300&display=swap');
+
+:root {
+    --purple: #3A233E;
+    --orange: #bf5701;
+    --light-orange: #e6a570;
+}
+
+body {
+    background-color: var(--purple);
+    font-family: "Montserrat", sans-serif;
+}
+
+.logo {
+    display: block;
+    margin: auto;
+    fill: var(--orange);
+}
+
+.container {
+    margin: auto;
+    margin-top: 5%;
+    width: 50%;
+    padding: 10px;
+    text-align: center;
+}
+
+h1 {
+    color: white;
+    font-weight: 500;
+    font-size: 3rem;
+}
+
+h2 {
+    color: white;
+    font-weight: 300;
+}
+
+#input1 {
+    width: 15em;
+    line-height: 20px;
+}
+
+
+.button-1 {
+    background-color: var(--orange);
+    border-radius: 8px;
+    border-style: none;
+    box-sizing: border-box;
+    color: white;
+    cursor: pointer;
+    display: inline-block;
+    font-family: "Montserrat", sans-serif;
```

Comment:
Is it possible to use the value "inherit" to always make it appear in the same font as the body?